### PR TITLE
blog(embertimes116): octane super rentals

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2019-09-20-the-ember-times-issue-116.md
Comment (ijlee2) [inline source/2019-09-20-the-ember-times-issue-116.md]:
```suggestion
If you are looking to contribute, check out the [super-rentals-tutorial repo on GitHub](https://github.com/ember-learn/super-rentals-tutorial)!
```

**Review by ijlee2 — APPROVED** [review]
Chris, thanks for the write-up. I didn't know the tutorial had been generated automatically, wow!

There's a CI failure for a past Times issue, unrelated to your changes. Do you want to address it now?

File: source/2019-09-20-the-ember-times-issue-116.md
Comment (amyrlam) [inline source/2019-09-20-the-ember-times-issue-116.md]:
```suggestion
[Godfrey Chan @chancancode](https://github.com/chancancode) and [Vaidehi Joshi @vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing content and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/) and check out the tutorial.
```

**Comment by amyrlam** [comment]
thanks for getting this in @chrisrng and for the review @ijlee2! 

i pushed a commit to fix the markdown linter. i think i need to rebase `amy/embertimes-116` after this is merged

File: source/2019-09-20-the-ember-times-issue-116.md
Comment (chrisrng) [reply source/2019-09-20-the-ember-times-issue-116.md]: I thought our conventions was to use the at github alias?

**Comment by chrisrng** [comment]
Thanks for fixing the build @amyrlam !

File: source/2019-09-20-the-ember-times-issue-116.md
Comment (amyrlam) [reply source/2019-09-20-the-ember-times-issue-116.md]: We've kinda been doing both lately e.g. https://blog.emberjs.com/2019/09/13/the-ember-times-issue-115.html. I guess someone did it and I thought it might be a bit more friendly

Comment (amyrlam) [reply source/2019-09-20-the-ember-times-issue-116.md]: What do you think? Def don't have to keep going this way

Comment (chancancode) [inline source/2019-09-20-the-ember-times-issue-116.md]: There is a good chance that https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/ would have been [filled in](https://github.com/ember-learn/super-rentals-tutorial/pull/55) by the time we send this. If it's up, it would be great to change the link to the introduction chapter instead!

Comment (chancancode) [reply source/2019-09-20-the-ember-times-issue-116.md]: Minor suggestion: maybe take out `and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots` in this first paragraph and combine it into the second one, so that the first paragraph is focused on content and the second one is focused on infrastructure.

File: source/2019-09-20-the-ember-times-issue-116.md
Comment (chancancode) [inline source/2019-09-20-the-ember-times-issue-116.md]:
```suggestion
If you are looking to contribute, or if you have any feedback on the content, check out the [super-rentals-tutorial repo on GitHub](https://github.com/ember-learn/super-rentals-tutorial)!
```

File: source/2019-09-20-the-ember-times-issue-116.md
Comment (amyrlam) [reply source/2019-09-20-the-ember-times-issue-116.md]: @chancancode ok, keep us posted. Are you saying that this link https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/ might be updated? I'm eyeing a post-4:30 PM pacific release

Comment (chancancode) [reply source/2019-09-20-the-ember-times-issue-116.md]: Yes! Working on it now!

Comment (chrisrng) [reply source/2019-09-20-the-ember-times-issue-116.md]: Cool will update the links in the blog

Comment (chrisrng) [reply source/2019-09-20-the-ember-times-issue-116.md]: Updated the text based on @chancancode 's comment and ill accept this (@amyrlam 's change) for the non-alias name

Comment (chrisrng) [reply source/2019-09-20-the-ember-times-issue-116.md]: Oh I guess I can't accept it now so i'll push in a  new commit :)

**Comment by chancancode** [comment]
@amyrlam @chrisrng Good news!

The page is live! https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/
The "finished" (part 1, at least) app is also continuously deployed to https://super-rentals-tutorial--ember-super-rentals.netlify.com/

**Comment by amyrlam** [comment]
@chancancode @chrisrng I think I made the change you want, feel free to suggest other changes / commit to the branch.

Will have to check out this tutorial, looking spiffy!

File: source/2019-09-20-the-ember-times-issue-116.md
Comment (chancancode) [inline source/2019-09-20-the-ember-times-issue-116.md]:
```suggestion
[Godfrey Chan @chancancode](https://github.com/chancancode) and [Vaidehi Joshi @vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing new tutorial content for Octane. You can now preview the [revamped Super Rentals tutorial](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/) in the [Octane guides](https://octane-guides-preview.emberjs.com/).
```

I think _maybe_ it would be nice to link to the Super Rentals app too, but it seems hard to fit it in, and the tutorial text already links you to it.

**Comment by chrisrng** [comment]
This LGTM if you're all good lets merge :)

**Pull request merged**
